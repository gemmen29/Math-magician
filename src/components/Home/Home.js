import React from 'react';
// import PropTypes from 'prop-types';
import styles from './Home.module.css';

const Home = () => (
  <section className={styles.home}>
    <h1 className={styles.home__header}>Welcome to our page</h1>
    <p className={styles.home__p}>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit ex atque
      numquam nobis at placeat mollitia id voluptas dolores, maxime alias est
      beatae ducimus laudantium quod iste error hic harum perferendis quae?
      Illum quas maxime earum, perspiciatis, accusamus, quod velit voluptatum
      aspernatur deleniti dignissimos adipisci vitae atque voluptatibus eaque
      incidunt!
    </p>
    <p className={styles.home__p}>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit ex atque
      numquam nobis at placeat mollitia id voluptas dolores, maxime alias est
      beatae ducimus laudantium quod iste error hic harum perferendis quae?
      Illum quas maxime earum, perspiciatis, accusamus, quod velit voluptatum
      aspernatur deleniti dignissimos adipisci vitae atque voluptatibus eaque
      incidunt!
    </p>
  </section>
);

Home.propTypes = {};

Home.defaultProps = {};

export default Home;
