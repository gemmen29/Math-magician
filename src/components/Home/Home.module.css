.home {
  display: flex;
  height: 100%;
  flex-direction: column;
  gap: 2.5em;
}

.home__header {
  font-size: 2rem;
  font-weight: bold;
}

.home__p {
  font-size: 1.2rem;
}
