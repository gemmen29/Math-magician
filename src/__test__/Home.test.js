import React from 'react';
import { render } from '@testing-library/react';
import Home from '../components/Home/Home';

describe('Test Home render', () => {
  test('Test Home', () => {
    const tree = render(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
