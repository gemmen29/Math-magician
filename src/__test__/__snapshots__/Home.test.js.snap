// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home render Test Home 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="home"
      >
        <h1
          class="home__header"
        >
          Welcome to our page
        </h1>
        <p
          class="home__p"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit ex atque numquam nobis at placeat mollitia id voluptas dolores, maxime alias est beatae ducimus laudantium quod iste error hic harum perferendis quae? Illum quas maxime earum, perspiciatis, accusamus, quod velit voluptatum aspernatur deleniti dignissimos adipisci vitae atque voluptatibus eaque incidunt!
        </p>
        <p
          class="home__p"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit ex atque numquam nobis at placeat mollitia id voluptas dolores, maxime alias est beatae ducimus laudantium quod iste error hic harum perferendis quae? Illum quas maxime earum, perspiciatis, accusamus, quod velit voluptatum aspernatur deleniti dignissimos adipisci vitae atque voluptatibus eaque incidunt!
        </p>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="home"
    >
      <h1
        class="home__header"
      >
        Welcome to our page
      </h1>
      <p
        class="home__p"
      >
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit ex atque numquam nobis at placeat mollitia id voluptas dolores, maxime alias est beatae ducimus laudantium quod iste error hic harum perferendis quae? Illum quas maxime earum, perspiciatis, accusamus, quod velit voluptatum aspernatur deleniti dignissimos adipisci vitae atque voluptatibus eaque incidunt!
      </p>
      <p
        class="home__p"
      >
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit ex atque numquam nobis at placeat mollitia id voluptas dolores, maxime alias est beatae ducimus laudantium quod iste error hic harum perferendis quae? Illum quas maxime earum, perspiciatis, accusamus, quod velit voluptatum aspernatur deleniti dignissimos adipisci vitae atque voluptatibus eaque incidunt!
      </p>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
